WORKSPACE

```diff
@@ -274,8 +274,14 @@ new_go_repository(
 )
 
 new_go_repository(
+    name = "io_k8s_apimachinery",
+    commit = "20e10d54608f05c3059443a6c0afb9979641e88d",
```

Comment:
you'll need to add the date comment to this line to pass our presumbit. I think this is `# Mar 31, 2017`.